modules/3d-tiles/src/utils/array-slice.js

```diff
@@ -0,0 +1,62 @@
+const typedArrayTypes = [
+  Int8Array,
+  Uint8Array,
+  Int16Array,
+  Uint16Array,
+  Int32Array,
+  Uint32Array,
+  Float32Array,
+  Float64Array
+];
+
+if (typeof ArrayBuffer !== 'undefined') {
+  typedArrayTypes.push(
+    Int8Array,
+    Uint8Array,
+    Int16Array,
+    Uint16Array,
+    Int32Array,
+    Uint32Array,
+    Float32Array,
+    Float64Array
+  );
+
+  if (typeof Uint8ClampedArray !== 'undefined') {
+    typedArrayTypes.push(Uint8ClampedArray);
+  }
+
+  // if (typeof CanvasPixelArray !== 'undefined') {
+  //   typedArrayTypes.push(CanvasPixelArray);
+  // }
+}
+
+/**
+ * Create a shallow copy of an array from begin to end.
+ *
+ * @param {Array} array The array to fill.
+ * @param {Number} [begin=0] The index to start at.
+ * @param {Number} [end=array.length] The index to end at which is not included.
+ *
+ * @returns {Array} The resulting array.
+ * @private
+ */
+export default function arraySlice(array, begin, end) {
```

Comment:
I wonder if we need this function, doesn't this look like a polyfill for really old browsers?